ansible/roles/chaos/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+# Chaos
+
+- name: Setup chaos directories
+  file: path="{{ item }}" owner=root group=root mode=0755 state=directory
+  with_items:
+    - "{{ chaos_dir }}"
+    - "{{ chaos_dir }}/start"
+    - "{{ chaos_dir }}/stop"
+  tags:
+    - start
+
+- name: Install chaos scripts
+  sudo: yes
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ chaos_dir }}/start/{{ item }}"
+    group: root
+    owner: root
+    mode: 0755
+    backup: yes
+  with_items: "{{ enable.split(',') }}"
+  tags:
+    - start
+
+- name: Start chaos
+  shell: "{{ chaos_dir }}/start/{{ item }}"
+  with_items: "{{ enable.split(',') }}"
+  tags:
+    - start
+
+- name: "Find stop scripts"
+  shell: "find {{ chaos_dir }}/stop -executable -type f"
+  register: stop_scripts
+  changed_when: false
+  failed_when: false
+  tags:
+    - debug_stop
+    - stop
+
+- name: Stop chaos
```

Comment:
I am not sure how we plan on using this, but are concerned at all about being locked out of the machine. At times in the past when cpu got too high we have been unable to ssh into machines, and since this relies on being able to reconnect to the machine and issues a series of commands to stop it, that seems like potentially a problem here.

Reply:
For this particular one I don't think tihs is an issue. It should only need to run one dd command .